docs/en/UI/AspNetCore/JavaScript-API/DOM.md

```diff
@@ -113,5 +113,4 @@ The following Bootstrap components and libraries are automatically initialized w
 
 * Tooltip
 * Popover
-* Timeage
-
+* Timeago
```

Comment:
ℹ️ I think this is the correct spelling but it's domain specific so someone should double-check me.